library/src/enabled/java/com/telefonica/tweaks/ui/TweakComponents.kt

```diff
@@ -179,48 +197,51 @@ fun TweakGroupBody(
 private fun ResetButton(
     onResetClicked: () -> Unit = {},
 ) {
-    Button(
+    TweakButton(
         onClick = onResetClicked,
-        modifier = Modifier.fillMaxWidth(),
-    ) {
-        Text("⚠️ Reset ⚠️")
-    }
+        text = "⚠️ Reset ⚠️",
+    )
 }
 
 @Composable
 fun TweakButton(entry: ButtonTweakEntry) {
-    Button(
+    TweakButton(
         onClick = entry.action,
-        modifier = Modifier.fillMaxWidth(),
-    ) {
-        Text(entry.name)
-    }
+        text = entry.name,
+    )
 }
 
 @Composable
 fun TweakNavigableButton(
     entry: RouteButtonTweakEntry,
     onClick: (String) -> Unit,
 ) {
-    Button(
+    TweakButton(
         onClick = { onClick(entry.route) },
-        modifier = Modifier.fillMaxWidth(),
-    ) {
-        Text(entry.name)
-    }
+        text = entry.name,
+    )
 }
 
 @Composable
 fun TweakNavigableButton(
     entry: CustomNavigationButtonTweakEntry,
     customNavigation: ((NavController) -> Unit) -> Unit,
 ) {
-    Button(
+    TweakButton(
         onClick = { customNavigation(entry.navigation) },
-        modifier = Modifier.fillMaxWidth(),
-    ) {
-        Text(entry.name)
-    }
+        text = entry.name,
+    )
+}
+
+@Composable
+fun TweakCategoryButton(
```

Comment:
do you think we can get rid of this TweakButton wrapper since it is not adding any new configuration to it?

Reply:
Yep, totally, you're right :)